content/en/docs/concepts/extend-kubernetes/_index.md

```diff
@@ -28,52 +28,50 @@ Kubernetes Project {{< glossary_tooltip text="Contributors" term_id="contributor
 find it useful as an introduction to what extension points and patterns exist, and their
 trade-offs and limitations.
 
-<!-- body -->
-
-## Overview
+Customization approaches can be broadly divided into [configuration](#configuration), which only
+involves changing command line arguments, local configuration files, or API resources; and [extensions](#extensions),
```

Comment:
nit: I like the content in the overview. Is it possible to break up the first sentence or create a list and then descriptions of the two types of customization approaches?

* configuration
* extensions

Reply:
This is existing text, I just added hyperlinks. I don't feel like I can write what you're imagining here @kbhawkey.

Could you use the Suggestion feature of GitHub or write the text you'd like to see?